src/_rules/shadow.js

```diff
@@ -7,6 +9,20 @@ const boxShadowDefaults = {
   xlarge: `0 9px 12px ${defaultColor}, 0 14px 28px ${defaultColor}`,
 };
 
+// TODO: rename to -s, -m, -l, -xs
 export const shadows = [
   [/^shadow-(small|medium|large|xlarge)$/, ([, size]) => ({ 'box-shadow': `var(--w-shadow-${size}, ${boxShadowDefaults[size]})` })],
-];
\ No newline at end of file
+];
+
+const dropShadowDefaultColor = '64, 64, 64';
+
+const dropShadowDefaults = {
+  s: [`drop-shadow(rgba(${dropShadowDefaultColor}, .16) 0 1px 6px) drop-shadow(rgba(${dropShadowDefaultColor}, .24) 0 1px 1px)`],
+  m: [`drop-shadow(rgba(${dropShadowDefaultColor}, .24) 0 3px 8px) drop-shadow(rgba(${dropShadowDefaultColor}, .16) 0 3px 6px)`],
+  l: [`drop-shadow(rgba(${dropShadowDefaultColor}, .23) 0 6px 8px) drop-shadow(rgba(${dropShadowDefaultColor}, .19) 0 10px 20px)`],
+  xl: [`drop-shadow(rgba(${dropShadowDefaultColor}, .22) 0 9px 12px) drop-shadow(rgba(${dropShadowDefaultColor}, .25) 0 14px 28px)`],
+};
+
+export const dropShadows = [
+  [/^drop-shadow-(s|m|l|xl)$/, ([, size]) => ({ 'filter': dropShadowDefaults[size] })],
```

Comment:
In the case od box-shadow we used the shadow tokens and applied defaults if those were missing. Here I see we set defaults only. I asked if that also should be the case here in https://github.com/warp-ds/component-classes/pull/57#discussion_r1202603931.

Reply:
What do you mean? Is this a blocker or can we iterate on this later?